include/seqan/graph_align/graph_algorithm_refine_exact_iterative.h

```diff
@@ -39,6 +39,8 @@
 #ifndef SEQAN_INCLUDE_SEQAN_GRAPH_ALGORITHM_REFINE_EXACT_ITERATIVE_H_
 #define SEQAN_INCLUDE_SEQAN_GRAPH_ALGORITHM_REFINE_EXACT_ITERATIVE_H_
 
+//#define SEQAN_DEBUG
+
```

Comment:
Am I right, that I need to
1) rename SEQAN_DEBUG to e.g. SEQAN_TCOFFEE_DEBUG
2) remove all these defines and to keep only the one in seqan_tcoffee.cpp
3) I have to place this define in my .cpp file *before* any includes (otherwise if I place it after, the flag is not defined in .h files)
?